src/ImageSharp/Image.FromBytes.cs

```diff
@@ -41,7 +41,7 @@ public static IImageFormat DetectFormat(Configuration configuration, byte[] data
         }
 
         /// <summary>
-        /// Reads the raw image information from the specified stream without fully decoding it.
+        /// Reads the raw image information from the given encoded byte array without fully decoding it.
```

Comment:
Are we also using `encoded` in other comments?

Reply:
For byte arrays, `IdentifyFormat` doesn't call them encoded in \<summary\> but `Load` does, so I went with the latter for consistency

Although the \<summary\> for streams/files doesn't mention `encoded` at all, so I can remove `encoded` from \<summary\> for byte arrays if preferable